src/vergil/main/Vergil.java

```diff
@@ -0,0 +1,107 @@
+package vergil.main;
+
+import vergil.components.Ui;
+import vergil.components.Storage;
+import vergil.components.Parser;
+import vergil.components.TaskList;
+
```

Comment:
I think it would be more suitable to split them into multiple sub-packages, as it would make the process of locating classes with strong associations easier.

I do agree, though, that some rearrangement is needed.